src/gui/processing/models/qgsmodeldesignerdialog.h

```diff
@@ -186,6 +184,9 @@ class GUI_EXPORT QgsModelDesignerDialog : public QMainWindow, public Ui::QgsMode
     void reorderOutputs();
     void setPanelVisibility( bool hidden );
     void editHelp();
+    void run();
+    void showPreviousResults( const QString &childId );
+    void showLog( const QString &childId );
```

Comment:
I feel the function names here aren't explicit enough -- what about showChildAlgorithmOutputs and showChildAlgorithmLog to make it explicit that this isn't about showing the logs or outputs of all algorithms? I know there's a big hint in the parameter name here :) still, I feel the functions name should bit a bit less generic.

Reply:
Good idea, done